plaso/parsers/aws_cloudtrail_log.py

```diff
@@ -0,0 +1,228 @@
+# -*- coding: utf-8 -*-
+"""JSON-L parser plugin for AWS CloudTrail log files.
+
+This parser is based on the log format documented at
+https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-event-reference-record-contents.html
+
+"""
+
+import json
+import os
+
+
+from dfdatetime import semantic_time as dfdatetime_semantic_time
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.parsers import manager
+from plaso.lib import errors
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.parsers import interface
+
+
+class AWSCloudTrailEventData(events.EventData):
+  """AWS CloudTrail log event data.
+
+  Attributes:
+    eventTime
```

Comment:
style nit: missing docstrings and docstring does not match the actual attribute names